hashicorp/boundary-ui
# Icu 14663 update manage dropdown for credential stores form

# Description
Updated the manage dropdown in the credential stores form to reflect HDS design.

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/14663) 

## Screenshots (if appropriate)\
**Before**
<img width="1799" alt="Credstore before1" src="https://github.com/user-attachments/assets/de7fe725-5139-4252-bcc6-e43177abf932">
<img width="1799" alt="Credstore before2" src="https://github.com/user-attachments/assets/2f237f6a-eb6b-4252-91a3-8c60ccfede84">
**After**
<img width="1799" alt="credstore after1" src="https://github.com/user-attachments/assets/d6c2eb19-2b61-4995-9ed2-8de8738b98f2">
<img width="1799" alt="credstore after2" src="https://github.com/user-attachments/assets/63eaa739-fc36-498c-8e92-cfd72db079b2">


## How to Test
1. Start up server/login to Boundary client
2. Navigate to orgs tab and select an org.
3. Navigate to projects in sidebar and select a project
4. Select credential store in sidebar and select credential store.
5. HDS change should be reflected.

## Checklist
~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
